lib/reporters/mini.js

```diff
@@ -77,6 +78,10 @@ MiniReporter.prototype.prefix = function (str) {
 MiniReporter.prototype._test = function (test) {
 	var title = test.title;
 
+	if (title.length > process.stdout.columns) {
```

Comment:
I am guessing `process.stdout.columns === undefined` during testing. Same reason as [here](https://github.com/sindresorhus/ava/issues/605#issue-138766792).

> This is because `tap` forks the process and does not emulate a tty environment in the child process.

You could force the issue in those tests simply by setting process.stdout.columns to some large number in the tests.